### ISSUE TITLE
performance on shapenet is inconsistent with the results in the paper

### ISSUE THREAD

**Issue by hualin95** [Zhang-VISLab/Learning-to-Segment-3D-Point-Clouds-in-2D-Image-Space#11]
Hi, Zhang, I find the both results(two choices) is lower than that in your paper(84.6 of Class mIouU and 88.8 of Instance IoU). Is there any detail I missed?

**Comment by Zhang-VISLab** [comment]
Hi there,

Thanks for your message. As we mentioned, the two released models are just simply retrained from scratch without any fine-tuning in order to show that the code is working. Recall that in our code, there are two parts: 2D image generation and Multiscale UNet. To reproduce the results in our paper, you have to fine-tune the approach on your side. Note that different 2D image representations will lead to different results, even the parameters in UNet are the same. Hope that this answers your question properly. Please do let us know if there is still any problem. Thanks.

**Comment by pyt1234** [comment]
@Zhang-VISLab Could you directly provide the checkpoint in your paper? I finetuned the provided checkpoint, but I cannot get the comparable results to the original ones. Thanks.

**Comment by YechengLyu** [comment]
> @Zhang-VISLab Could you directly provide the checkpoint in your paper? I finetuned the provided checkpoint, but I cannot get the comparable results to the original ones. Thanks.

Hello we do have the checkpoint trained using tensorflow-keras. Please check our readme file under Training choice 1: Training using pre-stored dataset - Demo model.

**Comment by pyt1234** [comment]
@YechengLyu Thanks. But I noticed that, for the Demo model, the performance is as follows:

> The demo model achieved 90.1% accuracy, 85.6% instance-mean-IoU and 83.1% class-mean-IoU on the test set.

But in your paper:

![image](https://user-images.githubusercontent.com/44813882/90972589-fc379e00-e54c-11ea-8297-aba691fcd83a.png)

The performance is not comparable. In README, you mentioned: 

> Notice: Both shared models were retrained from scratch, whose results are slightly different from the ones reported in our paper.

Why don't you provide the checkpoint that are NOT trained from scratch directly? (And I have tried to finetune as you said, but still the results can not be compared to the ones in your paper.)

**Comment by lluowoull** [comment]
> @YechengLyu Thanks. But I noticed that, for the Demo model, the performance is as follows:
> 
> > The demo model achieved 90.1% accuracy, 85.6% instance-mean-IoU and 83.1% class-mean-IoU on the test set.
> 
> But in your paper:
> 
> ![image](https://user-images.githubusercontent.com/44813882/90972589-fc379e00-e54c-11ea-8297-aba691fcd83a.png)
> 
> The performance is not comparable. In README, you mentioned:
> 
> > Notice: Both shared models were retrained from scratch, whose results are slightly different from the ones reported in our paper.
> 
> Why don't you provide the checkpoint that are NOT trained from scratch directly? (And I have tried to finetune as you said, but still the results can not be compared to the ones in your paper.)

The performance in this table looks weird, the class mIoU on the left does not equal to the average of each class IoU. If you compare their performance with PointCNN, they have several much lower classes, but the final mIoU is the same as PointCNN.

**Comment by Zhang-VISLab** [comment]
Dear all,

Thanks for your great comments. We are terribly sorry that we lost our CVPR 2020 code after submission. This repository is a reproduced work, and we released a pre-trained network model with 88.0% instance-mean-iou and 86.5% class-mean-iou. An updated ArXiv preprint is available. Could you please check our code to see whether there still exist some issues? If so, please open a new issue and we will try to resolve it asap. Thanks.
